src/_settings/mappings/career.json

```diff
@@ -0,0 +1,133 @@
+{
+  "properties": {
+    "active": {
+      "type": "boolean"
+    },
+    "applicant_types": {
+      "properties": {
+        "announcement_number": {
+          "type": "string"
+        },
+        "application_type": {
+          "properties": {
+            "active": {
+              "type": "boolean"
+            },
+            "applicant_type": {
```

Comment:
`applicant_type.applicant_type` seems odd way to structure the data. Should we change this to `name` or `title`?